src/codelens/flinkSqlProvider.test.ts

```diff
@@ -20,180 +22,302 @@ import {
   FlinkSqlCodelensProvider,
   getCatalogDatabaseFromMetadata,
   getComputePoolFromMetadata,
+  getDefaultCatalogDatabase,
 } from "./flinkSqlProvider";
 
 const testUri = Uri.parse("file:///test/file.sql");
 
-describe("codelens/flinkSqlProvider.ts FlinkSqlCodelensProvider", () => {
+describe("codelens/flinkSqlProvider.ts", () => {
   let sandbox: sinon.SinonSandbox;
-  let provider: FlinkSqlCodelensProvider;
-  let resourceManagerStub: sinon.SinonStubbedInstance<ResourceManager>;
-  let ccloudLoaderStub: sinon.SinonStubbedInstance<CCloudResourceLoader>;
-  let hasCCloudAuthSessionStub: sinon.SinonStub;
 
-  // NOTE: setting up fake TextDocuments is tricky since we can't create them directly, so we're
-  // only populating the fields needed for the test and associated codebase logic, then using the
-  // `as unknown as TextDocument` pattern to appease TypeScript
-  const fakeDocument: TextDocument = { uri: testUri } as unknown as TextDocument;
+  before(async () => {
+    await getTestExtensionContext();
+  });
```

Comment:
This `before` hook isn't necessary, but makes using the Extension Test Runner extension much happier since it doesn't run through the global [beforeAll](https://github.com/confluentinc/vscode/blob/09b206fcd180419e256a3b63291f57e1f2ae4d77/src/testing.ts#L56-L57) that sets extension context, checks the sidecar, etc.